recipe/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "python-ffmpeg" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4d5275a58da2a82fb550acbf496ff86e76d677e06b781baeefbe05a4cd47ab50
+
+build:
+  # ffmpeg is not available for s390x
+  skip: true  # [s390x]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - python
+    - pyee
+    - typing_extensions
+    - ffmpeg
+
+test:
+  requires:
+    - pip
+  imports:
+    - ffmpeg
```

Comment:
I would add just one simple test which actually calls ffmpeg binary just in case, if it turns out to be difficult at least test it on your local env

Reply:
definitively, since I've added the `ffmpeg` package now, even all tests make sense (there is the usual problem that `conftest.py` is not included in the tarball from pypi, therefore I will source everything from github archive).